src/persist/src/s3.rs

```diff
@@ -930,6 +930,15 @@ impl MinElapsed {
     }
 }
 
+// Make sure the "vendored" feature of the openssl_sys crate makes it into the
+// transitive dep graph of persist, so that we don't attempt to link against the
+// system OpenSSL library. Fake a usage of the crate here so that a good
+// samaritan doesn't remove our unused dep.
```

Comment:
🙈 sorry!

Reply:
it wasn't you! <3 (and anyway I don't blame anyone. I hope "good samaritan" came across as serious here, because it was)

Reply:
It does come across as serious! Just feel bad it's happened twice now.